website/docs/d/oracle_autonomous_database_regular.html.markdown

```diff
@@ -0,0 +1,156 @@
+
+* `subnet_id` - The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the subnet the resource is associated with.
+
+* `supported_regions_to_clone_to` - The list of regions that support the creation of an Autonomous Database clone or an Autonomous Data Guard standby database.
+
+* `tags` - A mapping of tags assigned to the Autonomous Databases.
+
+* `time_created` - The date and time the Autonomous Database was created.
+
+* `time_data_guard_role_changed` - The date and time the Autonomous Data Guard role was switched for the Autonomous Database. For databases that have standbys in both the primary Data Guard region and a remote Data Guard standby region, this is the latest timestamp of either the database using the "primary" role in the primary Data Guard region, or database located in the remote Data Guard standby region.
+
+* `time_deletion_of_free_autonomous_database` - The date and time the Always Free database will be automatically deleted because of inactivity. If the database is in the STOPPED state and without activity until this time, it will be deleted.
+
+* `time_local_data_guard_enabled_on` - The date and time that Autonomous Data Guard was enabled for an Autonomous Database where the standby was provisioned in the same region as the primary database.
+
+* `time_maintenance_begin` - The date and time when maintenance will begin.
+
+* `time_maintenance_end` - The date and time when maintenance will end.
+
+* `time_of_last_failover` - The timestamp of the last failover operation.
+
+* `time_of_last_refresh` - The date and time when last refresh happened.
+
+* `time_of_last_refresh_point` - The refresh point timestamp (UTC). The refresh point is the time to which the database was most recently refreshed. Data created after the refresh point is not included in the refresh.
+
+* `time_of_last_switchover` - The timestamp of the last switchover operation for the Autonomous Database.
+
+* `time_reclamation_of_free_autonomous_database` - The date and time the Always Free database will be stopped because of inactivity. If this time is reached without any database activity, the database will automatically be put into the STOPPED state.
+
+* `used_data_storage_size_in_gbs` - The storage space consumed by Autonomous Database in GBs.
+
+* `used_data_storage_size_in_tbs` - The amount of storage that has been used, in terabytes.
+
+* `vnet_id` - The ID to an Azure Resource Manager vnet resource.
+
+* `whitelisted_ips` - The client IP access control list (ACL). This feature is available for [Autonomous Database Serverless] (https://docs.oracle.com/en/cloud/paas/autonomous-database/index.html) and on Exadata Cloud@Customer. Only clients connecting from an IP address included in the ACL may access the Autonomous Database instance. If `arePrimaryWhitelistedIpsUsed` is 'TRUE' then Autonomous Database uses this primary's IP access control list (ACL) for the disaster recovery peer called `standbywhitelistedips`.
```

Comment:
we do not allow the term whitelisted in the provider, this must be changed to
```suggestion
* `allowed_ips` - The client IP access control list (ACL). This feature is available for [Autonomous Database Serverless] (https://docs.oracle.com/en/cloud/paas/autonomous-database/index.html) and on Exadata Cloud@Customer. Only clients connecting from an IP address included in the ACL may access the Autonomous Database instance. If `arePrimaryWhitelistedIpsUsed` is 'TRUE' then Autonomous Database uses this primary's IP access control list (ACL) for the disaster recovery peer called `standbywhitelistedips`.
```
or something else

Reply:
Done thanks.